dev_tools/pypath

```diff
@@ -21,6 +21,7 @@
 
```

Comment:
When i run this file 

```python
(cirq_venv) vtomole@vtomole:~/Cirq$ shellcheck dev_tools/pypath 

In dev_tools/pypath line 1:
# Copyright 2018 The Cirq Developers
^-- SC2148: Tips depend on target shell and yours is unknown. Add a shebang.
```

Reply:
The `dev_tools/pypath` should work with both bash and zsh.
We can check it using
```
$ shellcheck --shell=bash dev_tools/pypath
```

Here is how I test all files, `dev_tools/pypath` is added separately, because `file` does not identify it as a shell script:
```sh
$ git ls-files | xargs file | grep Bourne-Again.shell.script | cut -d: -f1 |
      xargs shellcheck --shell=bash dev_tools/pypath
```

Reply:
Can we add a shebang at the top so that it is identified as a shell script?

Reply:
Good point!  Done in c44b6bfb76788b5681603ee1984cac4986cb6c71.